benchmarks/buiter_et_al_2016_jsg/exp_2_low_resolution.prm

```diff
@@ -0,0 +1,30 @@
+#### This is the unstable wedge experiment 2
+#### This prm file is the same as the exp_2_high_resolution.prm file except changes below.
+
+include $ASPECT_SOURCE_DIR/benchmarks/buiter_et_al_2016_jsg/exp_2_high_resolution.prm
+
+
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement                 = 0
+  set Initial global refinement                   = 0    #The resolution is 2 mm/cell
+  set Time steps between mesh refinement          = 0
+end
+
+### Material properties
+# Using harmonic material averaging is required to achieve reasonable convergence 
+# behavior, particularly when the viscosity contrast is large.
+subsection Material model
+  set Model name                    = visco plastic
+
+  subsection Visco Plastic
+
+  
+  # Further testing revealed that decreasing the model resolution allows using a higher contrast between the minimum and maximum viscosity.
```

Comment:
remove the empty lines 22, 23; break the long lines to ones of approximately 80 characters